test/test.pluggableauthhandler.ts

```diff
@@ -159,7 +157,7 @@ describe('PluggableAuthHandler', () => {
       );
     });
 
-    it('should return executable response when successful', async () => {
+    it('should return SAML executable response when successful', async () => {
```

Comment:
Should have both an OIDC and SAML test without output_file specified where expiration time is not present in the executable response.